website/server/controllers/api-v3/coupon.js

```diff
@@ -11,12 +11,36 @@ import couponCode from 'coupon-code';
 let api = {};
 
 /**
+ * @apiDefine sudo access only
```

Comment:
Definitions usually use capital letters. Also, this will be rendered as `access only` since `Sudo` is the definition. It should be something like:

```
* @apiDefine Sudo Sudo Users
* Moderators with all access permissions.
```